### PR TITLE
A couple cargo nerfs via gun crate price increases while I wrap up my PR to port TG's roundstart cybernetic organs.

### PR DESCRIPTION
**Pull request by ghost**
Somebody buffed the shit out of the WT-550 and that was a terrible idea.

The WT-550 was prior to the change a worse stetchkin with more ammo that could take a bayonet.

20 damage, 20 rounds, multiple ammo types (like the stetchkin). 

As compared to the stetchkin, which did 25 damage but had less rounds.

Now the WT-550 got buffed out the ass-it fires 2 rounds, so 40 damage, and can be dual wielded to do up to 80 damage point blank. This puts it head and shoulders over the stetchkin and makes it obscenely powerful-it's good enough to go toe to toe with the C-20r nuclear operative weapon on the whole and it's cheap as shit to boot, almost everyone can have one. 

Instead of nerfing the WT-550 though, I just replaced the crate. Now you can order the WT-550 that boarding teams get, which has only a single burst and comes loaded with rubber rounds-lethal ammo now costs extra (unless you print more yourself which most people do.) So now it's reasonable again instead of being better than a traitor item.

Also I increased the cost of the combat shotgun crate to 30,000 and the riot shotgun crate to 8000. Why? Because anyone who does cargo can confirm that getting cargo credits is insanely simple.

Let's take a look at a common bounty for organs, for instance. If you butcher 3 monkeys, you get 20,000 credits-probably more if they're high priority-and that's enough for 2 combat shotgun crates and a WT crate, or 10 laser gun crates. That's obscene.

Now I'm of the opinion that since NT is energy and not ballistic weapon themed, and ballistics are the syndicate's specialty, any ballistics NT can field should be low-quality or highly expensive, and NT should instead have cheap energy weapons. Not to mention considering how powerful combat shotties are, it's worth the price increase.

Cargo has been due for a nerf for a while-due to the money and economy changes breaking the entire department which i've gone on rants about before, and frankly decreasing the sheer quantity of guns they can order will be a good change. Not that it matters too much since, you know, money is obscenely easy to come by-cargo on a bad round will still make 200,000$.

#### Changelog

:cl:  
rscadd: Added the WT occupying rifle crate to cargo, a single-shot WT rifle loaded with rubber bullets (can be swapped out)
rscdel: Removed the WT autorifle burst fire crate from cargo  
tweak: Adjusted the prices of ballistic weapons in cargo 
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -653,41 +653,41 @@
 	crate_name = "swat crate"
 
 /datum/supply_pack/security/armory/wt550_single
-	name = "WT-550 Auto Rifle Single-Pack"
-	desc = "Contains one high-powered, semiautomatic rifles chambered in 4.6x30mm. Requires Armory access to open."
+	name = "Surplus Security Autorifle Single-Pack"
+	desc = "Contains one high-powered, semiautomatic rifles chambered in 4.6x30mm rubber rounds. Requires Armory access to open."
 	cost = 2000
 	contains = list(/obj/item/gun/ballistic/automatic/wt550)
```
Comment (Bibby0110) [inline code/modules/cargo/packs.dm]: forgot this one didn't ya
```suggestion
	contains = list(/obj/item/gun/ballistic/automatic/wt550/occupying)
```

**Comment by maxion12345** [comment]
Don't you think this will mess a bit with balance during nukeops? Should the crew be forced to buy in heavily to cheap energy weapons due to bounties being nerfed and ballistic weapons being OP again, a syndicate could simply purchase an eshield and deflect all their cheap laser weapontry, while having to handle far less ballistic threat.
While I think the credit changes are a nice touch, the dual fire is going to be a non-issue with the dual-wield removal. Combined with a firing delay, I'd say it has less damage potential then any slug shotgun with that particular change.

Also this seems a very roundabout way to remove the dual burst under the guise of rp or someshit

**Comment by ghost** [comment]
@maxion12345 

Energy weapons are already the weapon of choice due to window spam at the moment.

If a nukie counters that, eh.

Also, again, cargo prices are insane. You can get 100K very easily, especially from the security budget card. I forsee no balance problems whatsoever. I'm not concerned. Crew handled nukies fine back in the day when war didn't even exist and all nukies ran was eshield-bulldog on blitz and cargo had no guns. Yog can handle it fine with how much everything has been powercrept.

Also, this wasn't the guise of RP or anything. This was just me removing the ability to order burst-fire WT 550 from cargo. 

Since Hopekz added the boarding rifle WT, then instead of me removing the burstfie from the WT and making the occupying force rifle an irrelevant distinction, I just made the occupying force rifle the standard one so I can use the other one as a loot reward or whatever. 

Also, I don't care about the dual wield removal and I doubt it'll go through.

**Comment by JamieD1** [comment]
@AdamElTablawy 

I feel 8k -> 30k is a little extreme. Maybe meet somewhere in the middle and see how it goes?

**Comment by Moja10** [comment]
I agree with Jamie

**Comment by Moja10** [comment]
On top of the fact that no PRs really adjust individual bounty prices

**Comment by ghost** [comment]
30k is fine.

**Comment by Bibby0110** [comment]
Do me a favor and explain what the occupying rifle is in the changelog for people reading that, (not everyone knows it's rubber bullets)

**Comment by JamieD1** [comment]
18k. 30k too much.

**Comment by Hopekz** [comment]
> I feel 8k -> 30k is a little extreme.

> 30k is fine.

> 18k. 30k too much.

Is this an episode of pawn stars?

**Comment by ghost** [comment]
I'm only obliging because I know you've been having a rough week and I don't want to argue with you.

**Pull request merged**